abstract.hbs

```diff
@@ -0,0 +1,11 @@
+<section id="abstract">
+  <p>
+  The purpose of this test suite is to demonstrate a path to interoperability
+  for the <a href="https://w3c.github.io/vc-data-integrity/">data integrity</a>
+  of ecdsa cryptosuites.
```

Comment:
You might want to link to the specs for the ecdsa suites this test suite covers such as: https://www.w3.org/TR/vc-di-ecdsa/

Reply:
Thanks, added [ecd9034](https://github.com/w3c-ccg/vc-di-ecdsa-test-suite/pull/1/commits/ecd90341e71062f20f4443ce11469f7fe6524f7d)